loop.go

```diff
@@ -83,7 +84,7 @@ func (c *Channel) IsAlive() bool {
 /**
 Close channel
 */
-func closeChannel(c *Channel, m *methods, args ...interface{}) error {
+func closeChannel(c *Channel, m *methods, reasons ...error) error {
```

Comment:
changing args ...interface{} to reasons ...error for clarity, it was receiving a error before as args ...interface.
And since there could be more than 1 error, I decided to keep the slice